bundles/org.openhab.binding.modbus.sunspec/src/main/java/org/openhab/binding/modbus/sunspec/internal/SunSpecConstants.java

```diff
@@ -58,6 +68,10 @@
     // Channel group ids
     public static final String GROUP_DEVICE_INFO = "deviceInformation";
     public static final String GROUP_AC_GENERAL = "acGeneral";
+    public static final String GROUP_AC_PHASE_A = "acPhaseA";
```

Comment:
Is that the naming used by the protocol specification? Otherwise I would prefer `L1`, `L2`, `L3`, which is the common naming.

Reply:
This specification use Phase A/B/C. For example here is one doc available: https://www.solaredge.com/sites/default/files/sunspec-implementation-technical-note.pdf see page 17.
But I understand your point as well. Maybe we could just change the labels?

Reply:
If A B C is the notation used by the standard this is probably how the devices are labeled, too. So we should keep that.

As you pointed out, we could use something like `A (L1)` for the labels.

Reply:
Thanks, rephased the group labels. Also found a typo I was overlooking so far.